testsuite/integration/web/web-enable-elytron.cli

```diff
@@ -1,6 +1,6 @@
 embed-server --server-config=standalone.xml
 
-/subsystem=undertow/application-security-domain=other:add(http-authentication-factory=application-http-authentication)
+#/subsystem=undertow/application-security-domain=other:add(http-authentication-factory=application-http-authentication)
```

Comment:
This line is not neccessary with current test implementation as 'application-security-domain' is configured via ServerSetupTask.

Actually whole custom cli script concept seems a little bit awkward in this context (I have to check whether we are running elytron profile in server setup task anyway). But I understand that in case there is no more configuration neccessary, it is quite nice way how to utilize existing tests also for Elytron testing.